docs/packaging-an-application/events-and-orchestration.md

```diff
@@ -27,6 +27,8 @@ application and are designed to be used to help manage the state of your applica
 when a container changes state. These events are published as soon as the Replicated operator reports that
 the container state has changed.
 
+When setting up event orchestration use unique event names and follow the rule of thumb that multiple children are fine but each container should have one parent. Event orchestration is critical for template functions and referenced containers should be started prior to the call of the template function.
```

Comment:
Yes, the point here is to ensure the reader knows that for aspects such as discovery of dynamically assigned ports those containers must be started first. There is an exception for template functions that return Replicated-planned information like NodePrivateIPAddressAll. I'll cleanup the language to make it more specific to dynamic content.